asciidoc/session-api.adoc

```diff
@@ -251,7 +324,7 @@ See <<Session configuration>> for more details.
 [[driver-async-transaction-fn]]
 === Transaction functions
 
-Transaction functions are the recommended form for containing transactional units of work.
+Transaction functions are *_the recommended form for containing transactional units of work_*.
```

Comment:
Based on recent highlighting of this issue, I think we could make this stand out more in either a TIP admonition, or a NOTE admonition?